src/cdk-experimental/menu/context-menu.ts

```diff
@@ -216,6 +216,7 @@ export class CdkContextMenuTrigger implements OnDestroy {
 
       this._contextMenuTracker.update(this);
       this.open({x: event.clientX, y: event.clientY});
+      this._menuPanel._menu?.focusFirstItem('mouse');
```

Comment:
I don't know that this should be mouse, since once _could_ trigger this event with the keyboard.

Reply:
Referring to the fact that this method handles the keyboard shortcut to trigger a context menu??

Reply:
Yes, exactly

Reply:
ok. As discussed, fixed to consider to clicked mouse button when setting focus